### DISCUSSION TITLE
list-image- [-url] adds space to the icon

### DISCUSSION THREAD

**Discussion by Tarshh** [tailwindlabs/tailwindcss#10935] [Help]
Hello,

I tried using the new list-image-[-url] class but it seems to add spacing to my icon on the bottom side? It does not have this extra spacing in the icon itself, it does that to every icon I have used for this case

When I use `list-disc` the list icon get centered like it should. See Image:
![image](https://user-images.githubusercontent.com/47401362/229611402-54e9cae0-a210-4e3e-ad67-acb0ef27e014.png)

Am I missing something?

![image](https://user-images.githubusercontent.com/47401362/229609604-df99d361-06bc-4c86-b647-bc02e23461ba.png)

The code that generates this is the following:

`
export function Checklist({ listItems }: Props) {
  return (
    <ul className="list-image-[url(/icons/check-yellow.svg)] pl-10">
      {listItems.map((item, i) => {
        return (
          <li key={i} className="mb-4 pl-4">
            {item}
          </li>
        );
      })}
    </ul>
  );
}
`

I can't seem to format the code well in here so I add another screenshot for the code:
![image](https://user-images.githubusercontent.com/47401362/229610664-7cb55290-48a9-4ac5-a960-17c9bc7b0591.png)

Using TailwindCss: 3.3.0

Thanks in advance.

**Comment by wongjn** [comment]
I believe that is simply how `list-style-image` "works". There seems to be quite limited positioning capabilities with it and `::marker`. It seems like the image is aligned along the baseline of the text, while the bounding box of the `::marker` is the text height. You can see how this varies by changing `font-size` and the `line-height` of the list.

**Reply by Tarshh** [reply]
Hmm, so no option to remove the extra height on the bottom?

Could check if line height could center it a bit more.
Otherwise the solution could be to use a flexbox with an image and a p tag to center them. But that is not very semantic...

Other reccomendations?

Thanks for the reply!

**Reply by yannickschuchmann** [reply]
@Tarshh you've found a good solution in the meantime? :)

**Comment by topherfangio** [comment]
For reference, in GitHub, you can format your code by surrounding it with 3 backticks. You can also specify the language to add appropriate syntax highlighting.

For example, the following

````
```jsx
export function Checklist({ listItems }: Props) {
  return (
    <ul className="list-image-[url(/icons/check-yellow.svg)] pl-10">
      {
        listItems.map((item, i) => {
          return ( <li key={i} className="mb-4 pl-4"> {item} </li> );
        })
      }
    </ul>
  );
}
```
````

will render like this:

```jsx
export function Checklist({ listItems }: Props) {
  return (
    <ul className="list-image-[url(/icons/check-yellow.svg)] pl-10">
      {
        listItems.map((item, i) => {
          return ( <li key={i} className="mb-4 pl-4"> {item} </li> );
        })
      }
    </ul>
  );
}
```

_Note: I'm not 100% sure if this is valid JSX syntax, but I'm just trying to show you how to format it_ 😄

You can learn more in the docs: https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks
